styx-scheduler-service/src/test/java/com/spotify/styx/StyxSchedulerServiceFixture.java

```diff
@@ -244,14 +247,6 @@ void givenWorkflowEnabledStateIs(Workflow workflow, boolean enabled) {
     }
   }
 
-  void givenTheGlobalEnableFlagIs(boolean enabled) {
```

Comment:
Is there a test on system level that tests that things are not scheduled when this is disabled?

Reply:
No, there were no tests that verified that things are not scheduled if the global enable flag is disabled.